src/operator/tensor/matrix_op-inl.h

```diff
@@ -257,6 +257,29 @@ struct TransposeParam : public dmlc::Parameter<TransposeParam> {
   }
 };
 
+
+template<typename xpu, typename DType>
```

Comment:
Can you add comments explaining input params

Reply:
Added!